Sources/LiveViewNative/ViewTree.swift

```diff
@@ -87,6 +89,13 @@ struct ViewTreeBuilder<R: RootRegistry> {
             view
         }
     }
+
+    @ViewBuilder
+    private func applyTag(element: ElementNode, to view: some View) -> some View {
+        let tag = element.attributeValue(for: "tag")
+
+        view.tag(tag)
```

Comment:
Could this modifier be conditionally applied like `id` is? I'm worried that applying it to every element even if it isn't specified could cause some unexpected behaviors with nested elements.

It'd still have to ensure the tag's type is `Optional<String>` though.

```suggestion
        if let tag = element.attributeValue(for: "tag") {
            view.tag(Optional<String>.some(tag))
        } else {
            view
        }
```